BUILD.md

```diff
@@ -418,6 +418,27 @@ onnxruntime_perf_test
 onnxruntime_test_all
 ```
 
+#### Build Instructions(Jetson Nano)
+
+1. Build ACL Library (skip if already built)
+```
+cd ~
+git clone https://github.com/Arm-software/ComputeLibrary.git
```

Comment:
This is an example for someone who wants to build ACL. If they have a pre-built version, they may use it as mentioned above.